internal/verifier/sigstore/sigstore.go

```diff
@@ -49,7 +49,7 @@ func New(trustedRoot, accessToken, cacheDir string) (*Sigstore, error) {
 		return nil, err
 	}
 	sev, err := verify.NewSignedEntityVerifier(trustedMaterial, verify.WithSignedCertificateTimestamps(1),
-		verify.WithTransparencyLog(1), verify.WithOnlineVerification())
```

Comment:
Is this because the API changed between versions, or is there a different reason for it?

Reply:
This one is an API change 👍

Reply:
Yea, we've separated requiring a number of transparency log entries from timestamps. Using the new WithObserverTimestamps API states that you require at least one timestamp from any source (the preferred API to use).